index.html

```diff
@@ -0,0 +1,240 @@
+            <li><img src="../Portfolio-finish-mobile-version/img/circle.svg" alt=""></li>
+            <li><p>2015</p></li>
+          </ul>
+          </div>
+          <p>
+            A daily selection of privately
+            personalized reads; no accounts
+            or sign-ups required.
+          </p>
+          <ul class="languages">
+            <li><a href="">html</a></li>
+            <li><a href="">css</a></li>
+            <li><a href="">javascript</a></li>
+            <li><a href="">Ruby on rails</a></li>
+          </ul>
+          <div class="rev">
+            <button><a href="" class="btn">See Project</a></button>
+          </div>
+        </div>
+      </li>
+      <li class="item tonic">
+        <img src="../Portfolio-finish-mobile-version/img/work3.svg" alt="">
+        <div>
+          <div>
+          <h1>Tonic</h1>
+          <ul class="canopy">
+            <li><h3>CANOPY</h3></li>
+            <li><img src="../Portfolio-finish-mobile-version/img/circle.svg" alt=""></li>
+            <li><p>Back End Dev</p></li>
+            <li><img src="../Portfolio-finish-mobile-version/img/circle.svg" alt=""></li>
+            <li><p>2015</p></li>
+          </ul>
+          </div>
+          <p>
+            A daily selection of privately
+            personalized reads; no accounts
+            or sign-ups required.
+          </p>
+          <ul class="languages">
+            <li><a href="">html</a></li>
+            <li><a href="">css</a></li>
+            <li><a href="">javascript</a></li>
+            <li><a href="">Ruby on rails</a></li>
+          </ul>
+          <div class="rev">
+            <button><a href="" class="btn">See Project</a></button>
+          </div>
+        </div>
+      </li>
+      <li class="item multi-post-stories">
+        <img src="../Portfolio-finish-mobile-version/img/work4.svg" alt="">
+        <div>
+          <div>
+          <h1>Multi-Post Stories</h1>    
+          <ul class="canopy">
+            <li><h3>CANOPY</h3></li>
+            <li><img src="../Portfolio-finish-mobile-version/img/circle.svg" alt=""></li>
+            <li><p>Back End Dev</p></li>
+            <li><img src="../Portfolio-finish-mobile-version/img/circle.svg" alt=""></li>
+            <li><p>2015</p></li>
+          </ul>
+          </div>
+          <p>
+            A daily selection of privately
+            personalized reads; no accounts
+            or sign-ups required.
+          </p>
+          <ul class="languages">
+            <li><a href="">html</a></li>
+            <li><a href="">css</a></li>
+            <li><a href="">javascript</a></li>
+            <li><a href="">Ruby on rails</a></li>
+          </ul>
+          <div class="rev">
+            <button><a href="" class="btn">See Project</a></button>
+          </div>
+        </div>
+      </li>
+    </ul>
+    </main>
+  </section>
+  <aside class="about">
+    <div class="abt_head">
+    <h1>About <br>
+        Myself</h1>
```

Comment:
- Please, let's kindly use another header tag like `h2-h6` as this is not the main header on your page and a website should only have one h1 which denotes the main header of the webpage.